content/docs/2.10/concepts/admission-webhooks.md

```diff
@@ -0,0 +1,22 @@
++++
+title = "Admission Webhook"
+description = "Automatically validate resource changes to prevent misconfiguration and enforce best practices"
+weight = 100
++++
+
+## Admission Webhooks
+
+> 💡 The Admission Webhooks are an opt-in feature and will become an opt-out feature as of KEDA v2.12.
```

Comment:
Let's remove this:
```suggestion
> 💡 The Admission Webhooks are an opt-in feature and will become an opt-out feature as of KEDA v2.12.
```

Because:
![image](https://user-images.githubusercontent.com/4345663/210947967-deeb556b-8ab0-4b21-abe6-0f1fcee97013.png)

Reply:
nice!